src/Language/DifferentialDatalog/Validate.hs

```diff
@@ -557,11 +557,9 @@ exprValidate1 _ _ ctx EVar{..} | ctxInRuleRHSPositivePattern ctx
                                           = return ()
 exprValidate1 d _ ctx (EVar p v)          = do _ <- checkVar p d ctx v
                                                return ()
-exprValidate1 d _ _   (EApply p fnames as) = do
-    fs <- concat <$> mapM (\fname -> checkFuncs p d fname (length as)) fnames
-    check d (length fs > 0) p $ "Unknown function '" ++ (nameLocalStr $ head fnames) ++ "'"
-    check d (length fs == 1 || (not $ null as)) p $ "Ambiguous function name '" ++ (nameLocalStr $ head fnames) ++ "' may refer to:\n  " ++
-                                                    (intercalate "\n  " $ map (\f -> name f ++ " at " ++ spos f) fs)
+exprValidate1 _ _ _   EApply{}            = return ()
```

Comment:
why isn't this checked anymore?

Reply:
I am guessing this moved to typechecking

Reply:
yes, and also to `exprValidate1 EFunc{}`